apps/staking/src/app/api/publishers-ranking/route.tsx

```diff
@@ -0,0 +1,10 @@
+import { NextResponse } from "next/server";
+
+export async function GET() {
+  const publisherRankingsResponse = await fetch(
+    "https://www.pyth.network/api/publishers-ranking?cluster=pythnet",
```

Comment:
This endpoint is just a thin proxy to web-api, see https://github.com/pyth-network/pyth-web/blob/main/pages/api/publishers-ranking.tsx.  We should just fetch to `https://web-api.pyth.network/publishers_ranking?cluster=pythnet` instead to avoid the extra hop.

Also make sure to add `{ cache: 'no-store' }` or [next will cache the response](https://nextjs.org/docs/app/api-reference/functions/fetch#optionscache) and since there's no datestamp in the query params, we'd get stale ranking data in the UI.

Note that web-api itself is also fairly slim and is really just hitting a redis cache [or calling a pretty simple clickhouse query](https://github.com/pyth-network/web-api/blob/main/web_api/db.py#L1035-L1064) so IMO we should ideally move away from using web-api too and just wrap this up in a node lib, but we'll need to get access control to clickhouse from Vercel set up so that's a problem for another day......

Reply:
used it in rewrite